pkgs/stdenv/cross/default.nix

```diff
@@ -1,35 +1,48 @@
-{ lib, allPackages
+{ lib
 , system, platform, crossSystem, config
 }:
 
-rec {
-  vanillaStdenv = (import ../. {
-    inherit lib allPackages system platform;
+let
+  bootStages = import ../. {
+    inherit lib system platform;
     crossSystem = null;
-    # Ignore custom stdenvs when cross compiling for compatability
+    # Remove config.replaceStdenv to ensure termination.
```

Comment:
Ooops, old comment should still be there, bad copy paste from `custom/default.nix`